db/tasks.php

```diff
@@ -28,10 +28,10 @@
     array(
         'classname' => '\logstore_xapi\task\emit_task',
         'blocking' => 0,
-        'minute' => '*/5',
+        'minute' => '*/1',
```

Comment:
I saw now that you can customize it here:
http://yourmoodle.com/admin/tool/task/scheduledtasks.php?action=edit&task=logstore_xapi%5Ctask%5Cemit_task

not sure if it will override the db/tasks.php setting, but I assume so. I'll give it a try.